ui/widgets/tracklist.go

```diff
@@ -592,6 +597,13 @@ func (t *Tracklist) onShowContextMenu(e *fyne.PointEvent, trackIdx int) {
 		})
 		unfavorite.Icon = myTheme.NotFavoriteIcon
 		t.ctxMenu.Items = append(t.ctxMenu.Items, playlist, download)
+		if !t.Options.HideSharing {
```

Comment:
Same comment as the Album page here - always create the menu item, and just disable it if sharing is not enabled. (change the disable condition below to `len(t.selectedTracks()) != 1 || t.Options.HideSharing`

I suppose the tracklist option should be renamed to `DisableSharing` as well.